### ISSUE TITLE
Release checksums are using full file path

### ISSUE THREAD

**Issue by rzanetti-cpqd** [flameshot-org/flameshot#2878]
### Flameshot Version

flameshot-12.1.0-1.ubuntu-20.04.amd64.deb

### Installation Type

Using the ready-made package from Github Releases

### Operating System type and version

Ubuntu 20.04.4 LTS (Focal Fossa)

### Description

Hello
I believe there is extra unwanted information on the releases` checksum files.

<hash> <dir>/<file> 
as opposed to
<hash> <file>

This breaks automated setups that put the file and its checksum on the same directory and run a check to validate its consistency.
The workaround is quite simple, as I can crop out the path before checking the sum, but I thought it would be valid to report, as from my understanding this is an unexpected setup.

Thanks

### Steps to reproduce

Take a checksum file, for instance [flameshot-12.1.0-1.ubuntu-20.04.amd64.deb.sha256sum](https://github.com/flameshot-org/flameshot/releases/download/v12.1.0/flameshot-12.1.0-1.ubuntu-20.04.amd64.deb.sha256sum)
Check the file path after the hash

### System Information

Ubuntu 20.04.4 LTS (Focal Fossa)

**Comment by mmahmoudian** [comment]
yes, I think it is a good suggestion. It is related to the following lines:

- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L199
- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L289
- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L298
- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L444
- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L510
- https://github.com/flameshot-org/flameshot/blob/master/.github/workflows/Linux-pack.yml#L567
